tools/mypy_helpers/remove_unneeded_type_ignores.py

```diff
@@ -14,12 +15,14 @@ def main() -> int:
             subprocess.check_call(
                 (
                     "sed",
-                    "-i",
+                    # https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux
+                    "-i/tmp/bogus.bak",
```

Comment:
this should get replaced with python if we intend anyone to use it -- I just added it as a quick devscript for me to use and didn't really intend anyone to use it.

I'd also recommend adding gnu coreutils / findutils / sed to your path personally so you don't have the broken bsd toolset